docs/index.rst

```diff
@@ -0,0 +1,12 @@
+python-cloudant documentation
+=============================
+
+This is the official Cloudant client library for Python.
+
+.. toctree::
+   :maxdepth: 4
```

Comment:
I see the toctree maxdepth varies from place to place, and they appear to have been chosen carefully so the generated TOC does look good. However, I didn't see a generated index anywhere, is this toctree level what is used for the search? In which case is there any reason not to go deeper than 4?